model/otel/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: otel.sdk.span.processor.queue_capacity
+    stability: development
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.processor.spans_processed
+    type: metric
+    metric_name: otel.sdk.span.processor.spans_processed
+    stability: development
+    brief: "The number of spans for which the processing has finished, either successful or failed"
+    note: |
+      For successful processing, `error.type` must be empty. For failed processing, `error.type` must contain the failure cause.
+      For the SDK Simple and Batching Span Processor a span is considered to be processed already when it has been submitted to the exporter, not when the corresponding export call has finished.
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching Span Processors MUST use `queue_full` for spans dropped due to a full queue.
+        examples: ["queue_full"]
+
+  - id: metric.otel.sdk.span.exporter.spans_inflight
+    type: metric
+    metric_name: otel.sdk.span.exporter.spans_inflight
+    stability: development
+    brief: "The number of spans which were passed to the exporter, but that have not been exported yet (neither successful, nor failed)"
+    note: |
+      For successful exports, `error.type` must be empty. For failed exports, `error.type` must contain the failure cause.
+    instrument: updowncounter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
+      - ref: otel.sdk.component.name
+
+  - id: metric.otel.sdk.span.exporter.spans_exported
+    type: metric
+    metric_name: otel.sdk.span.exporter.spans_exported
+    stability: development
+    brief: "The number of spans for which the export has finished, either successful or failed"
+    note: |
+      For successful exports, `error.type` must be empty. For failed exports, `error.type` must contain the failure cause.
+      For exporters with partial success semantics (e.g. OTLP with `rejected_spans`), rejected spans must count as failed and only non-rejected spans count as success.
+      If no rejection reason is available, `rejected` SHOULD be used as value for `error.type`.
+    instrument: counter
+    unit: "{span}"
+    attributes:
+      - ref: otel.sdk.component.type
```

Comment:
I think we should include recommended `server.address` and `server.port` attributes on exporter metrics. It's good to know where you are sending data to.

Reply:
Those would not apply to all exporters (e.g. stdout).  My thinking is that we should encourage using protocol-level instrumentation (e.g. http/gRPC) for details like this.

Reply:
Kind of agree with @dashpole here. I don't think this belongs in this metric.
Nonetheless, I think it would make sense to add `exporter.request.*` metrics to track request stats (e.g. bytes sent, response codes, server details). However, I don't think that this should happen in this PR, but rather in a separate, follow-up PR. It is an enhancement to gain more fine grained insights in addition to the metrics to this PR, but doesn't have an impact on them.

Reply:
> Those would not apply to all exporters (e.g. stdout).

not a problem, just add them with requirement level `recommended: when applicable`. We do include these attributes on logical operations across semconv, so they do belong here.

Reply:
Added in [830edfb](https://github.com/open-telemetry/semantic-conventions/pull/1631/commits/830edfb15972eb6ff2c7475996f1bcd3fec340ba).

However, for attribute references for metrics there is no schema.compliant `recommended: when applicable` IINM.
I added `note: recommended when applicable` instead, please let me know if this is the correct approach.